ui/src/components/ResultBlock/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export const ResultBlock: React.FC<{ result : any }> = ({ result }) => { 
+	return <>
+		<div className="ResultBlock">{ result.failure ? result.failure.message : "passed" }</div>
+	</>
+};
```

Comment:
I'm not sure whats going on with this indentation, it doesn't look like this in my IDE

Reply:
Same for the rest of the files